Cargo.toml

```diff
@@ -26,7 +26,8 @@ serde = "1.0"
 
 [dev-dependencies]
 opentelemetry-jaeger = { version = "0.11" }
-tokio = { version = "0.2", default-features = false, features = ["rt-core", "blocking", "time", "stream"] }
+opentelemetry = { version = "0.12", default-features = false, features = ["trace", "metrics", "tokio-support"] }
+tokio = { version = "1", default-features = false, features = ["time"] }
```

Comment:
we shouldn't need this dependency at all anymore as the `tokio-support` features of opentelemetry should handle this for us.